1-documentation/doc-styles.css

```diff
@@ -0,0 +1,36 @@
+@import "../../for-import/main.css";
+
+.wrapper-doc {
+    text-align: center;
+    margin: 0;
+    padding: 0;
+}
+
+.embed-wrapper {
+    width: 100%;
+    height: auto;
+    margin: 2rem auto 1rem auto;
```

Comment:
an alternative to this `margin: 2rem auto 1rem;` 
this works like `top`, `right and left`, `bottom`